### PR TITLE
Document default `API ROOT` within each api urls files (ref: `apiurls.py` and `urls.py`)

### LINKED ISSUE #489: No module named `qtimeseries.apiurls`
### Checklist

- [X] I've searched through the existing [issues](https://github.com/g3w-suite/g3w-admin/issues) and this bug has never been reported before

### Subject of the issue

After performing a clean install I've noticed that the following warning is displayed within the docker log:

```
No module named 'qtimeseries.apiurls'
```

### Steps to reproduce

1. follow the installation instructions in here: [g3w-suite-docker/README_DEV.md](https://github.com/g3w-suite/g3w-suite-docker/blob/9475d8260979309e9e583a951306326b016247ea/README_DEV.md)
2. and remember to update `G3WSUITE_LOCAL_CODE_PATH` path accordingly to your local G3W-SUITE code location. 🎗️ 😜

### Environment

- g3w-admin: __v3.5.dev-20230322091536__
- operating system: __Windows 11 (64 bit)__

### Link to your project

_No response_

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by Raruto**
## List of changes

- add a small hint to third party developers about default `API ROOT` value at the top of each `urls.py` and `apiurls.py` file
- replace verbose header metadata information, with a new variable: `__license__ = "MPL 2.0"`
- removed `# coding=utf-8` (but please, restore it if you think it is necessary)
- apply same comment styling for "nested" section comments, eg:
```py
#############################################################
# I am section
############################################################# 

some code here..

```
- ☢️ refactor file `base/urls.py`

Closes: https://github.com/g3w-suite/g3w-admin/issues/489
Supersedes: https://github.com/g3w-suite/g3w-admin/pull/490

### CODE REVIEW

**Comment by wlorenzetti** [comment]
@Raruto check my updates, thanks.

File: g3w-admin/qdjango/urls.py [hunk 1]
```diff
@@ -54,12 +67,15 @@
         name='qdjango-project-download'
     ),
 
-    # for ajaxFiler
-    re_path(r'^(?P<group_slug>[-_\w\d]+)/projects/fast/update/(?P<slug>[-_\w\d]+)/$',
-        login_required(QdjangoProjectFastUpdateView.as_view()), name='qdjango-project-fast-update'),
+    # For fast project update
+    # TODO: try to reactivate
+    #re_path(r'^(?P<group_slug>[-_\w\d]+)/projects/fast/update/(?P<slug>[-_\w\d]+)/$',
+    #    login_required(QdjangoProjectFastUpdateView.as_view()), name='qdjango-project-fast-update'),
```
Comment (Raruto) [inline g3w-admin/qdjango/urls.py:73]: image

## Docker log

```log
2023-03-27 09:54:02 Internal Server Error: /en/qdjango/buildings/projects/
2023-03-27 09:54:02 Traceback (most recent call last):
2023-03-27 09:54:02   File "/usr/local/lib/python3.10/dist-packages/django/core/handlers/exception.py", line 34, in inner
2023-03-27 09:54:02     response = get_response(request)
2023-03-27 09:54:02   File "/usr/local/lib/python3.10/dist-packages/django/core/handlers/base.py", line 145, in _get_response
2023-03-27 09:54:02     response = self.process_exception_by_middleware(e, request)
2023-03-27 09:54:02   File "/usr/local/lib/python3.10/dist-packages/django/core/handlers/base.py", line 143, in _get_response
2023-03-27 09:54:02     response = response.render()
2023-03-27 09:54:02   File "/usr/local/lib/python3.10/dist-packages/django/template/response.py", line 106, in render
2023-03-27 09:54:02     self.content = self.rendered_content
2023-03-27 09:54:02   File "/usr/local/lib/python3.10/dist-packages/django/template/response.py", line 83, in rendered_content
2023-03-27 09:54:02     content = template.render(context, self._request)
2023-03-27 09:54:02   File "/usr/local/lib/python3.10/dist-packages/django/template/backends/django.py", line 61, in render
2023-03-27 09:54:02     return self.template.render(context)
2023-03-27 09:54:02   File "/usr/local/lib/python3.10/dist-packages/django/template/base.py", line 171, in render
2023-03-27 09:54:02     return self._render(context)
2023-03-27 09:54:02   File "/usr/local/lib/python3.10/dist-packages/django/template/base.py", line 163, in _render
2023-03-27 09:54:02     return self.nodelist.render(context)
2023-03-27 09:54:02   File "/usr/local/lib/python3.10/dist-packages/django/template/base.py", line 937, in render
2023-03-27 09:54:02     bit = node.render_annotated(context)
2023-03-27 09:54:02   File "/usr/local/lib/python3.10/dist-packages/django/template/base.py", line 904, in render_annotated
2023-03-27 09:54:02     return self.render(context)
2023-03-27 09:54:02   File "/usr/local/lib/python3.10/dist-packages/django/template/defaulttags.py", line 312, in render
2023-03-27 09:54:02     return nodelist.render(context)
2023-03-27 09:54:02   File "/usr/local/lib/python3.10/dist-packages/django/template/base.py", line 937, in render
2023-03-27 09:54:02     bit = node.render_annotated(context)
2023-03-27 09:54:02   File "/usr/local/lib/python3.10/dist-packages/django/template/base.py", line 904, in render_annotated
2023-03-27 09:54:02     return self.render(context)
2023-03-27 09:54:02   File "/usr/local/lib/python3.10/dist-packages/django/template/defaulttags.py", line 212, in render
2023-03-27 09:54:02     nodelist.append(node.render_annotated(context))
2023-03-27 09:54:02   File "/usr/local/lib/python3.10/dist-packages/django/template/base.py", line 904, in render_annotated
2023-03-27 09:54:02     return self.render(context)
2023-03-27 09:54:02   File "/usr/local/lib/python3.10/dist-packages/django/template/defaulttags.py", line 312, in render
2023-03-27 09:54:02     return nodelist.render(context)
2023-03-27 09:54:02   File "/usr/local/lib/python3.10/dist-packages/django/template/base.py", line 937, in render
2023-03-27 09:54:02     bit = node.render_annotated(context)
2023-03-27 09:54:02   File "/usr/local/lib/python3.10/dist-packages/django/template/base.py", line 904, in render_annotated
2023-03-27 09:54:02     return self.render(context)
2023-03-27 09:54:02   File "/usr/local/lib/python3.10/dist-packages/django/template/defaulttags.py", line 446, in render
2023-03-27 09:54:02     url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
2023-03-27 09:54:02   File "/usr/local/lib/python3.10/dist-packages/django/urls/base.py", line 90, in reverse
2023-03-27 09:54:02     return iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs))
2023-03-27 09:54:02   File "/usr/local/lib/python3.10/dist-packages/django/urls/resolvers.py", line 677, in _reverse_with_prefix
2023-03-27 09:54:02     raise NoReverseMatch(msg)
2023-03-27 09:54:02 django.urls.exceptions.NoReverseMatch: Reverse for 'qdjango-project-fast-update' not found. 'qdjango-project-fast-update' is not a valid view function or pattern name.
```

File: g3w-admin/base/urls.py
Comment (Raruto) [inline g3w-admin/base/urls.py:1]: @wlorenzetti please double check too (I hope I didn't break many things here..)

I rewrite this file in order to make it a little bit "more" clear the difference between `apiurls.py` and `urls.py` routes

Comment (wlorenzetti) [reply g3w-admin/base/urls.py:1]: Every tests pass, so I think you didn't break nothing :) 
Thanks.

**Pull request merged**